bin/update_deps.sh

```diff
@@ -16,7 +16,7 @@
 
 set -e
 
-UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+UPDATE_BRANCH=${UPDATE_BRANCH:-"release-15"}
```

Comment:
I checked other branches, maybe release-1.15 is right?

Reply:
Fixed, thanks for catching!